releasenotes/notes/2.2/prepare-2.2.0-99d722b656fba0a8.yaml

```diff
@@ -0,0 +1,28 @@
+---
+prelude: >
+    The Qiskit v2.2 release further increases the functionality and versatility of the SDK
+    including enhancements for the C API and the transpiler as well as many other improvements
+    and bug fixes. Some of these additions include:
+
+      * **C API transpile function:** This version of Qiskit has added the ability for users to
+        call a version of the :func:`.transpile` function directly through the C API. The
+        :cpp:func:`qk_transpile` behaves similarly to the preset pass managers in Python,
+        with some exceptions that are not relevant in the C context. A collection of standalone
+        transpiler passes have also been added to the C API.
+      
+      * **Litinski transformation pass:** A new transformation pass was added to deal with circuits
+        with Clifford and single qubit RZ-rotation gates. The implementation is based on the transformation
+        described in `arXiv:1808.02892 <https://arxiv.org/abs/1808.02892>`__.
```

Comment:
Do we want to mention that this algorithm is important for fault tolerant compilation?

Reply:
Added in 582d48df49ba8e5c8ed22ca0c09de5095c2238d9